tensorflow/core/kernels/data/prefetch_dataset_op.cc

```diff
@@ -197,11 +193,11 @@ class PrefetchDatasetOp::Dataset : public DatasetBase {
         TF_RETURN_IF_ERROR(WriteStatus(writer, i, buffer_element.status));
         if (buffer_element.status.ok()) {
           TF_RETURN_IF_ERROR(writer->WriteScalar(
-              full_name(strings::StrCat("buffer[", i, "].size")),
+              full_name(strings::StrCat(kBuffer, "[", i, "].size")),
```

Comment:
make ".size" be a named constant `kSizeSuffix` and similarly for ".code" and ".error_message"